app/resources/add_note_base.html.template

```diff
@@ -15,25 +15,11 @@
 {% if not markdup %}
   <input type="hidden" name="add_note" value="yes">
   <input type="hidden" name="id" value="{{params.id}}">
-{% endif %}
+{% endif %} 
 
-<div class="fields-table note" role="complementary">
-  <div class="header section">
-    <h1>
-      {% if not markdup %}
-        {% if person.full_name %}
-          {% blocktrans with person.full_name as full_name %}Status of {{full_name}} {% endblocktrans %}
-        {% else %}
-          {% trans "Status of Person" %}
-        {% endif %}
-      {% else %}
-        {% trans "Mark records as duplicate" %}
-      {% endif %}
-    </h1>
-  </div>
+<div class="status section">
   {% if not markdup %}
-    <div class="section">
-      <h2>Status</h2>
+    <center><h2>Status</h2></center>
```

Comment:
Define a CSS rule in css-default.template to centerize h2 element instead of using ```<center>```. Make it a rule for e.g., ".fields-table h2" so that it won't affect h2 elements in other places.

In principle, HTML file should only contain the content and its structure. And CSS file should define the look-and-feel. In that principle, usage of tags like ```<center>```, ```<font>```, etc. are discouraged.